### ISSUE TITLE
TypeORM module resolution issue after 0.2.42

### ISSUE THREAD

**Issue by spalt08** [tonivj5/typeorm-naming-strategies#16]
TypeORM has recently released the 0.2.42 version and added an `exports` field into `package.json`.

The `snake-naming.strategy.js` file currently imports the `typeorm/util/StringUtils` module from the `typeorm` package, which conflicts with the new `package.json` `exports` field.

```
Error: Cannot find module '/var/app/current/node_modules/typeorm/util/StringUtils'
   ...
  code: 'MODULE_NOT_FOUND',
  path: '/var/app/current/node_modules/typeorm/package.json'
}
```

**Comment by spacebluecompany** [comment]
I'm having same issue !

**Comment by giladgd** [comment]
@spalt08 @spacebluecompany This issue was just fixed in `typeorm@0.2.43` (https://github.com/typeorm/typeorm/pull/8660), update the `typeorm` version used in your project and try again
